azavea/pfb-network-connectivity
# Update census tile color ramp to deciles

## Overview

Sets the color ramp for census block connectivity to 10 equal segments (rather than nine 6%-wide segments and one 46% top segment).

### Demo

Before:
![image](https://user-images.githubusercontent.com/6598836/52568768-b2112080-2ddd-11e9-902d-602683a05238.png)

After:
![image](https://user-images.githubusercontent.com/6598836/52568772-b50c1100-2ddd-11e9-964f-72bc1fb0eef0.png)

## Notes

- This PR is pointed at the branch for #707, since that changes the style setup.  I'll rebase after that's merged.
- I updated `census_blocks.mss` as well. I'm still of two minds about whether to keep that, but while it's here it seems better to have it in sync than not.

## Testing Instructions

Rebuild and restart your Tilegarden container, then load a neighborhood and view the "Census blocks with access" layer.  Everything should look a bit more dire.

Resolves #700.
